adam/axis.py

```diff
@@ -1,7 +1,6 @@
-from attr import attrs, attrib, evolve
-from attr.validators import instance_of
-
 from adam.utilities import sign
+from attr import attrib, attrs, evolve
```

Comment:
Sorry for all the import diff spam. I asked PyCharm to optimize imports without remembering that I don't think we do `isort` on this repository.